send.go

```diff
@@ -35,17 +35,13 @@ const (
 	WebhookURLOrgWebhookPrefix = "https://example.webhook.office.com"
 )
 
-// DisableWebhookURLValidation is a special keyword used to indicate to
-// validation function(s) that webhook URL validation should be disabled.
-const DisableWebhookURLValidation string = "DISABLE_WEBHOOK_URL_VALIDATION"
-
```

Comment:
Making it deprecated would make the client's code unreliable (thinking it would deactivate validation while this in fact doesn't do anything...). I think it would be better to introduce a breakable change for this one.

Reply:
Ok, best of two worlds I guess: to make the client's code reliable, I just added a simple case in `API.ValidateWebhook()` method and the related deprecated public function to support `DisableWebhookURLValidation` which is now a deprecated const.